Demo/wwwroot/js/custom.login.js

```diff
@@ -14,15 +18,15 @@ async function handleSignInSubmit(event) {
     // send to server for registering
     let makeAssertionOptions;
     try {
-        var res = await fetch('/assertionOptions', {
-            method: 'POST', // or 'PUT'
-            body: formData, // data can be `string` or {object}!
-            headers: {
-                'Accept': 'application/json'
-            }
-        });
-
-        makeAssertionOptions = await res.json();
+        // use jquery ajax instead of fetch because of safari browser and platform authenticator
```

Comment:
Safari seems to have a pretty good support for fetch: https://caniuse.com/?search=fetch.
Do we really need to use ajax here?